plugins/europeana/record.js

```diff
@@ -202,9 +202,9 @@ export default (axiosOverrides) => {
       const concepts = (edm.concepts || []).map(reduceEntity).map(Object.freeze);
       const places = (edm.places || []).map(reduceEntity).map(Object.freeze);
       const agents = (edm.agents || []).map(reduceEntity).map(Object.freeze);
-      const timespans = (edm.timespans || []).map(reduceEntity).map(Object.freeze);
+      const times = (edm.timespans || []).map(reduceEntity).map(Object.freeze);
 
-      const entities = [].concat(concepts, places, agents, timespans)
+      const entities = [].concat(concepts, places, agents, times)
```

Comment:
Revert this change, as here we use agents and concepts (as opposed to people and topics) so should also use timespans.